package.json

```diff
@@ -1,12 +1,30 @@
 {
-	"name": "AeroGearServicesWeb",
+	"name": "raincatcher",
```

Comment:
copy/paste error?

Reply:
I do not care as much as this code may be reverted. I wanted to get some decision made on typescript vs plain JS vs Webpack  and have it close to PR.